content/en/docs/setup/independent/install-kubeadm.md

```diff
@@ -198,24 +198,24 @@ systemctl enable kubelet && systemctl start kubelet
 Install CNI plugins (required for most pod network):
 
 ```bash
-CNI_VERSION="v0.6.0"
+export CNI_VERSION="v0.6.0"
```

Comment:
This isn't necessary, I am pretty sure. Setting the variable is sufficient ; you don't need to export it.